### PR TITLE
[Merged by Bors] - Delete unused modules in fluvio-sc

### PR DESCRIPTION
**Pull request by nicholastmosher**
All of these modules in `fluvio-sc` were completely unused, i.e. they were not linked in the module tree and deleting them has no effect on the project. Let me know if any of this should be saved, but if not I think it's a good cleanup.

### CODE REVIEW

**Review by sehz — CHANGES_REQUESTED** [review]

File: crates/fluvio-sc/src/tests/TESTS.md [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-* SPU status should still be up since SC didn't have change to receive status from SPU
-* Bring back up SC
-* SPU status should be change to offline after health check resync period
-
-## Bring up 2 SPU
-
-* Bring up SC and SPU
-* Create 2 custom SPU
-* 2nd SPU status is initially empty
-* After while, 2nd SPU's status should be offline since we didn't bring up 2nd SPU yet
```
[the remark this answers is not in the document]
Comment (nicholastmosher) [reply crates/fluvio-sc/src/tests/TESTS.md:28]: It was in a directory where everything else was unused and got deleted, so I figured it was no longer relevant.

[the remark this answers is not in the document]
Comment (sehz) [reply crates/fluvio-sc/src/tests/TESTS.md:28]: this needs to be kept

[the remark this answers is not in the document]
Comment (nicholastmosher) [reply crates/fluvio-sc/src/tests/TESTS.md:28]: Ok, this file is no longer deleted in this PR

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**